_posts/2020-12-30-PC-Engines-Firmware-v4-13-0-3.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.13.0.3"
+date:   2021-01-27
```

Comment:
@Sakoram it will be rather 29th. The blog post is published later when validation is done.

Reply:
fixed